scipy/stats/_qmc.py

```diff
@@ -1140,36 +1140,55 @@ class LatinHypercube(QMCEngine):
        integration and visualization." Statistica Sinica, 1992.
     .. [8] B. Tang, "Orthogonal Array-Based Latin Hypercubes."
        Journal of the American Statistical Association, 1993.
+    .. [9] Susan K. Seaholm et al. "Latin hypercube sampling and the
+       sensitivity analysis of a Monte Carlo epidemic model".
+       Int J Biomed Comput, 23(1-2), 97–112,
+       :doi:`10.1016/0020-7101(88)90067-0`, 1988.
 
     Examples
     --------
-    Generate samples from a Latin hypercube generator.
+    In [9]_, a Latin Hypercube sampling strategy was used to sample a
+    parameter space to produce a sensitivity analysis on the parameters of
+    an epidemic model.
+
+    The epidemic model had 6 parameters representing 4 contact probabilities,
+    the probbility of illness and the probability of withdrawal.
```

Comment:
Eliminated "model" for PEP8 line length. Numbers less than 10 are usually spelled out. 

I misread this the first time. Without the comma, it can be read as though "the probability of illness" and "the probability of withdrawal" are supposed to be the "four contact probabilities", which wouldn't make sense. Switching the order makes it impossible to misread.
```suggestion
    The six parameters of the model represented the probability of illness, 
    the probability of withdrawal, and four contact probabilities,
```